### PR TITLE
chore: update version to alpha.7

### PR DESCRIPTION
**Pull request by jelbourn**
R: @kara @hansl @robertmesserle

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,54 @@
...
+* **radio:** only emit change event if native input does. ([#911](https://github.com/angular/material2/issues/911)) ([23a61ab](https://github.com/angular/material2/commit/23a61ab)), closes [#791](https://github.com/angular/material2/issues/791)
+* **slide-toggle:** fix runtime exception for incorrect mousedown binding. ([#828](https://github.com/angular/material2/issues/828)) ([bbbc87f](https://github.com/angular/material2/commit/bbbc87f)), closes [#828](https://github.com/angular/material2/issues/828)
+* fix type mismatches when offline compiling. ([#835](https://github.com/angular/material2/issues/835)) ([4bb7790](https://github.com/angular/material2/commit/4bb7790))
+
+
+### Features
+
+* Add NgModules ([#950](https://github.com/angular/material2/issues/950)) ([ca351b2](https://github.com/angular/material2/commit/ca351b2))
+
+Note that NgModules are not the supported way of including the Material components in your app.
```
Comment (robertmesserle) [inline CHANGELOG.md]: not => now

File: src/components/button-toggle/package.json [hunk 2]
```diff
@@ -23,7 +23,7 @@
   },
   "homepage": "https://github.com/angular/material2#readme",
   "peerDependencies": {
-    "@angular2-material/core": "2.0.0-alpha.6-3",
+    "@angular2-material/core": "2.0.0-alpha.7",
     "@angular/forms": "^0.2.0"
```
Comment (kara) [inline src/components/button-toggle/package.json]: I think this should be ^0.3.0

File: src/components/checkbox/package.json [hunk 3]
```diff
@@ -21,7 +21,7 @@
   },
   "homepage": "https://github.com/angular/material2#readme",
   "peerDependencies": {
-    "@angular2-material/core": "2.0.0-alpha.6-3",
+    "@angular2-material/core": "2.0.0-alpha.7",
     "@angular/forms": "^0.2.0"
```
Comment (kara) [inline src/components/checkbox/package.json]: 0.3.0

File: src/components/button-toggle/package.json (hunk 2 above)
Comment (jelbourn) [inline src/components/button-toggle/package.json]: Good catch, fixed.

File: src/components/checkbox/package.json (hunk 3 above)
Comment (jelbourn) [inline src/components/checkbox/package.json]: Fixed.

**Pull request merged**